tidb-server/main.go

```diff
@@ -323,12 +327,30 @@ func validateConfig() {
 		log.Error("TiDB run with skip-grant-table need root privilege.")
 		os.Exit(-1)
 	}
+	if _, ok := config.ValidStorage[cfg.Store]; !ok {
+		nameList := ""
```

Comment:
There is a `strings.Join` in Go standard library.

See https://godoc.org/strings#Join